packages/animations/lib/src/open_container.dart

```diff
@@ -2,14 +2,16 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+// ignore_for_file: avoid_as
+
```

Comment:
empty line not used. Plus the space above is using a linter which I don't think would apply to anything since line 6 has nothing

Reply:
This comment asks the analyzer to ignore `avoid_as` throughout this file.

Reply:
We're still looking for better ways to ignore `avoid_as`.

Reply:
Done; please see comment above.